website/.vuepress/footer.ts

```diff
@@ -11,6 +11,7 @@ export const cnFooter = `
           <div style="text-align: center;">
             <h3>文档</h3>
             <a style="display: block; color: rgb(209, 213, 219); margin-bottom: 10px;" href="/bitsail/zh/documents/start/index.html">快速开始</a>
+            <a style="display: block; color: rgb(209, 213, 219); margin-bottom: 10px;" href="/bitsail/zh/documents/faq/README.md">FAQ</a>
```

Comment:
I think you should replace '.md' with '.html'.
And  you can test it locally (see https://bytedance.github.io/bitsail/zh/community/contribute.html#%E8%AF%B7%E6%B1%82%E4%BB%A3%E7%A0%81%E5%AE%A1%E6%9F%A5)

Reply:
get it！